### DISCUSSION TITLE
Shortcuts for markdown-editor

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#4235] [Feature Requests]
First of all amazing work with the new Markdown editor. Really love it!
To improve the user-experience I'd suggest to add shortcuts for the options. 

It could be comparably to WordPress (any many other md editors):
### Ctrl / Command Keys
* b ... Bold
* i ... Italic
* u ... Underline
* k ... Insert/edit link

### Alt + Shift / Ctrl + Option
* m ... Insert Image
* 1 ... Heading 1
* 2 ... Heading 2
* 3 ... Heading 3
* 4 ... Heading 4
* 5 ... Heading 5
* 6 ... Heading 6
* 7 ... Text ("reset" option - also not available from the editors option

**Comment by rijkvanzanten** [comment]
I wish I could upvote this more than once 😄

**Reply by benhaynes** [reply]
I helped with that... 😉

**Reply by rijkvanzanten** [reply]
We should be able to call the `edit` function:

https://github.com/directus/directus/blob/301d5a3337cc7634e08ebf39ff18328bf8dbaacd/app/src/interfaces/markdown/markdown.vue#L204

We should be able to re-use the `useShortcut` composable: https://github.com/directus/directus/blob/main/app/src/composables/use-shortcut/use-shortcut.ts

@Dominic-Marcelino wanna open a PR for this? I _think_ it might be as simple as 

```ts
useShortcut('meta+b', () => edit('bold'), codemirrorEl);
useShortcut('meta+i', () => edit('italic'), codemirrorEl);
```

but it'll need some testing

**Comment by rijkvanzanten** [comment]
I believe this was since implemented
